libs/shared/common-services/src/validation.service.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { TestBed } from '@angular/core/testing';
+import { ValidationService } from './validation.service';
+import { take } from 'rxjs/operators';
+
+describe('ValidationService', () => {
+  let service: ValidationService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ValidationService],
+    });
+
+    service = TestBed.inject(ValidationService);
+  });
```

Comment:
we should use the setup() function strategy for initializing the service, for test isolation
see how the other tests are written and make the setup function please